### PR TITLE
Portfolio projects

### PR DESCRIPTION
**Pull request by codewithed**
Completed the following actions for portfolio-projects section:
- add images of projects
- created sections for each project
- add about me section

### CODE REVIEW

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @codewithed , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
The Pull Request is following the GitHub Flow :heavy_check_mark: 
The PR title and description are professional and descriptive  :heavy_check_mark: 
Linters are working properly  :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -22,19 +18,117 @@
             <h1>I'm Edmund<br> Glad to see you</h1>
```
[the remark this answers is not in the document]
Comment (mwanawabangona) [reply index.html:18]: - [ ] `required`
- [ ] please make the changes requested by the previous reviewer

File: index.html [hunk 2]
```diff
@@ -22,19 +18,117 @@
...
+                    <li class="back-end">Back End Dev</li>
+                    <li class="date">2015</li>
+                </ul>
+                <p class="main-text">A daily selection of privately <br>
+                personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li class="small-button">html</li>
+                    <li class="small-button">css</li>
+                    <li class="small-button">javascript</li>
+                </ul>
```
[the remark this answers is not in the document]
Comment (mwanawabangona) [reply index.html:84]: - [ ] `required`
- [ ] please make the changes requested by the previous reviewer here

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @codewithed  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 


## Highlights
* ✅  Good job implement some of the requested changes :heavy_check_mark:


## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Div685 — APPROVED** [review]
Hi, **@codewithed**, I'm Divyesh your reviewer for this PR :man_technologist: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Keep on coding! :muscle:

## Highlights
- :heavy_check_mark: Good job implementing the requested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Code review by **Divyesh Patel**.

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.)_

------
>_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**